client/signup/jetpack-connect/plans.jsx

```diff
@@ -156,16 +147,11 @@ const Plans = React.createClass( {
 							steps={ 3 } />
 
 						<div id="plans" className="plans has-sidebar">
```

Comment:
yeah, all those css classes were added before the new lint check for their formatting... I think we should address those warnings in a separate PR to not make this more complex and (potentially) style modifying